src/app/fdctl/run/run.c

```diff
@@ -398,17 +204,26 @@ main_pid_namespace( void * args ) {
   if( FD_UNLIKELY( sched_getaffinity( 0, sizeof(cpu_set_t), floating_cpu_set ) ) )
     FD_LOG_ERR(( "sched_getaffinity failed (%i-%s)", errno, fd_io_strerror( errno ) ));
 
-  pid_t child_pids[ FD_TILE_MAX + 1 ];
-  char  child_names[ FD_TILE_MAX + 1 ][ 32 ];
+  pid_t child_pids[ FD_TOPO_MAX_TILES+1 ];
+  char  child_names[ FD_TOPO_MAX_TILES+1 ][ 32 ];
+  struct pollfd fds[ FD_TOPO_MAX_TILES+2 ];
+
+  fds[ 0 ] = (struct pollfd){ .fd = args->pipefd[ 1 ], .events = 0 };
```

Comment:
You can move this element to the end of the array to eliminate all the `1+`s below